randovania/gui/dialog/multiplayer_session_browser_dialog.py

```diff
@@ -164,12 +161,11 @@ def update_list(self):
             self.item_model.setItem(i, 0, name)
             self.item_model.setItem(i, 1, is_user_in_session)
             self.item_model.setItem(i, 2, join_date)
-            self.item_model.setItem(i, 3, state)
-            self.item_model.setItem(i, 4, num_users)
-            self.item_model.setItem(i, 5, num_worlds)
-            self.item_model.setItem(i, 6, has_password)
-            self.item_model.setItem(i, 7, creator)
-            self.item_model.setItem(i, 8, creation_date)
+            self.item_model.setItem(i, 3, num_users)
+            self.item_model.setItem(i, 4, num_worlds)
+            self.item_model.setItem(i, 5, has_password)
+            self.item_model.setItem(i, 6, creator)
+            self.item_model.setItem(i, 7, creation_date)
```

Comment:
```py
for col, item in enumerate((
    name,
    is_user_in_session,
    join_date,
    num_users,
    num_worlds,
    has_password,
    creator,
    creation_date,
):
    self.item_model.setItem(i, col, item)
```
so that we don't need to update magic indices?